src/nldumpbytes.c

```diff
@@ -40,7 +40,7 @@ void nl_dump_bytes(uintptr_t offs, const uint8_t *bytes, size_t num)
     {
         const size_t thisGo = (num >= 16 ? 16 : num);
 
-        printf("%08lX  ", offs);
+        printf("%08lX  ", (unsigned long int)offs);
```

Comment:
Any reason to not do?

printf(PRIxPTR "  ", offs);

Reply:
None whatsoever.  I blame sleep deprivation.